packages/editor-ui/src/views/SettingsApiView.vue

```diff
@@ -61,7 +61,8 @@ export default defineComponent({
 			return this.settingsStore.isPublicApiEnabled;
 		},
 		isRedactedApiKey(): boolean {
-			return this.apiKey.includes('*');
+			if (!this.apiKeys) return false;
+			return this.apiKeys[0].apiKey.includes('*');
```

Comment:
We should have a computed for this. It would simplify things quite a bit. I.e.

```ts
apiKey(): {
  return Array.isArray(this.ApiKeys) ? this.apiKeys[0] : undefined
}
```

Reply:
Very good point. I did not care much about the FE because, in the next phase, I'm going to nuke the whole screen!